spec/controllers/queue_items_controller_spec.rb

```diff
@@ -3,19 +3,17 @@
 describe QueueItemsController do
   describe "GET index" do
     context "with user logged in" do
-
       it "has a @queueitems variable" do
         current_user = Fabricate(:user)
         session[:user_id] = current_user.id
         queueitem = Fabricate(:queue_item, user: current_user)
         queueitem2 = Fabricate(:queue_item, user: current_user)
         get :index
-        expect(assigns(:queue_items)).to eq([queueitem,queueitem2])
+        expect(assigns(:queue_items)).to match_array([queueitem,queueitem2])

```

Comment:
You should use `queue_item` instead - in ruby, 2 different words as variables or method names should be snake_cased.